images/builder/docker/custom-docker-builder/Dockerfile

```diff
@@ -14,7 +14,7 @@
 #
 # The standard name for this image is openshift/origin-custom-docker-builder
 #
-FROM openshift/origin-base
+FROM openshift/origin-base:v1.5
```

Comment:
Do you have the ability to overwrite `FROM`? Maybe when you push to dist-git? As a rule we don't use versioned `FROM` in these `Dockerfiles`